README.md

```diff
@@ -88,6 +89,12 @@ List of the variables:
 
 ![console_gif](./screenshots/consoleGif.gif)
 
+## Firebase Storage
+
+If you see any references to `gs://` or `bucketDomain` anywhere the app may use Firebase Storage. By including the storage bucket you can also try to perform list, download, upload, delete operations on Firebase Storage.
+
+![firebase_strage](./screenshots/firebasestorage.png)
```

Comment:
small typo, you forgot the o  in _storage_  :^) 
(i know i'm being annoying lol sorry)

also, the screenshot disclose your bucket id, so don't forget do delete it when you're done testing.
it's for testing purposes i guess, but still I thought it's worth saying(in case they charge you or something).